src/components/LanguageButtons.js

```diff
@@ -9,31 +9,29 @@ const App = () => {
     i18n.changeLanguage(lng);
   };
 
-  //the es button is not working, but the zh one is. For es, I get the error: "missingKey es translation"
+  //the es button is not working, but the zh and en-US ones are. For es, I get the error: "missingKey es translation"
 
   return (
     <GridContainer>
-      <Grid col={8} offset={9}>
-        <Grid row gap>
-          <Grid>
-            <Button
-              type="button"
-              unstyled
-              onClick={() => changeLanguage("en-us")}
-            >
-              English
-            </Button>
-          </Grid>
-          <Grid>
-            <Button type="button" unstyled onClick={() => changeLanguage("es")}>
-              Spanish
-            </Button>
-          </Grid>
-          <Grid>
-            <Button type="button" unstyled onClick={() => changeLanguage("zh")}>
-              Chinese
-            </Button>
-          </Grid>
+      <Grid row gap>
+        <Grid>
+          <Button
+            type="button"
+            unstyled
+            onClick={() => changeLanguage("en-us")}
+          >
+            English
+          </Button>
+        </Grid>
+        <Grid>
+          <Button type="button" unstyled onClick={() => changeLanguage("es")}>
+            Spanish
```

Comment:
I would suggest replacing the button text in this section with the language names in their actual languages (e.g. Español instead of Spanish)